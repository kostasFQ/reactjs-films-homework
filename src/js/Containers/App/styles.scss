body {
  padding: 0;
  margin: 0;
  min-width: 830px;
}

#APP {
  display: flex;
  flex-direction: column;
}
