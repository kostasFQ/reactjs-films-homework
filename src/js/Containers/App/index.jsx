import { hot } from 'react-hot-loader';
import React from 'react';
import './styles.scss';
import Header from '../Header';
import MovieDetailsPage from '../MovieDetailsPage';
import image from '../../../imgs/cover-image.png';

const fakeFilm = {
  pict: image,
  title: 'The jungle book',
  genres: ['Adventure', 'Drama', 'Family', 'Fantasy'],
  duration: '1h 46m',
  rating: 4.8,
  preview: 'There are growing dangers in the wizarding world of 1926 New York.  Something mysterious is leaving a path of destruction in the streets, threatening to expose the wizarding',
};

const App = () => (
  <div>
    <Header />
    <MovieDetailsPage {...fakeFilm} />
  </div>
);

export default hot(module)(App);
