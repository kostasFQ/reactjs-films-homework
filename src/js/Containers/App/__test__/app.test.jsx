import React from 'react';
import renderer from 'react-test-renderer';
import App from '../index';

test('test component render', () => {
  const render = renderer.create(
    <App />,
  );
  const component = render.toJSON();

  expect(component).toMatchSnapshot();
  expect(component.type).toBe('div');
});
