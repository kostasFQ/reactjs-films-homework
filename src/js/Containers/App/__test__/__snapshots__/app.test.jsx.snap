// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component render 1`] = `
<div>
  <div
    className="container"
  >
    <h1
      className="title"
    >
      FILMS
    </h1>
    <div>
      <input
        className="input"
        placeholder="search"
        type="text"
      />
    </div>
  </div>
  <div>
    <img
      alt="pic"
      className="poster"
      src="test-file-stub"
    />
    <div
      className="container"
    >
      <div
        className="div"
      >
        <div
          className="container"
        >
          <h1
            className="h1"
          >
            THE JUNGLE BOOK
          </h1>
          <ul
            className="list"
          >
            <li
              className="item"
            >
              Adventure
            </li>
            <li
              className="item"
            >
              Drama
            </li>
            <li
              className="item"
            >
              Family
            </li>
            <li
              className="item"
            >
              Fantasy
            </li>
            <span>
              1h 46m
            </span>
          </ul>
          <div
            className="container"
          >
            <div
              className="whiteStars"
            />
            <div
              className="fillStars"
              style={
                Object {
                  "width": "96px",
                }
              }
            />
            <div
              className="ratingBox"
            >
              4.8
            </div>
          </div>
        </div>
        <div
          className="container"
        >
          <div
            className="buttonsBlock"
          >
            <button
              className="container"
              onClick={null}
              type="button"
            >
              Watch Now
            </button>
            <div
              className="x"
            >
              <button
                className="container"
                onClick={[Function]}
                type="button"
              >
                View Info
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
